src/ansiblelint/rules/galaxy.py

```diff
@@ -15,18 +14,17 @@
 
 
 class GalaxyRule(AnsibleLintRule):
-    """Rule for checking collection version is greater than 1.0.0 and checking for changelog."""
+    """Rule for checking collections."""
 
     id = "galaxy"
-    description = "Confirm via galaxy.yml file if collection version is greater than or equal to 1.0.0 and check for changelog."
+    description = "Confirm that collection's units are valid."
     severity = "MEDIUM"
     tags = ["metadata"]
     version_added = "v6.11.0 (last update)"
```

Comment:
Should `version_added` be updated for `GalaxyRule`? I wasn't entirely sure if this attribute just meant when the rule was first introduced. Even then, for `GalaxyVersionIncorrectRule`, I just used a version I'd imagined that the rule will be released under.

Reply:
I think `version_added` typically reflects the original introduced version - @Qalthos or @ssbarnea can you confirm?